components/Map.tsx

```diff
@@ -1,29 +1,29 @@
 import * as React from "react";
-import { useState, useEffect, useRef, useCallback } from "react";
+import { useCallback, useEffect, useRef, useState } from "react";
 // eslint-disable-next-line import/named
+import { gql, useQuery } from "@apollo/client";
+import Col from "react-bootstrap/Col";
 import ReactMapGL, { MapEvent, NavigationControl } from "react-map-gl";
```

Comment:
The linter is failing here now. I believe the comment above `// eslint-disable-next-line import/named` needs to be directly above this import.